inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/RudderParsingRules.scala

```diff
@@ -178,24 +177,6 @@ object RudderCpuParsing extends FusionReportParsingExtension with Loggable {
 }
 
 /**
-* <CFKEY>
-*/
-class RudderPublicKeyParsing(keyNormalizer:PrintedKeyNormalizer) extends FusionReportParsingExtension {
-  override def isDefinedAt(x:(Node,InventoryReport)) = { x._1.label == "CFKEY" }
-  override def apply(x:(Node,InventoryReport)) : InventoryReport = {
-    optText(x._1) match {
-      case None => x._2
-      case Some(key) =>
-        keyNormalizer(key) match {
-          case "" => x._2 //we have an empty key !
-          case k => x._2.copy( node = x._2.node.copy( publicKeys = (new PublicKey(key) +: x._2.node.publicKeys ) ) )
-
-        }
-    }
-  }
-}
-
-/**
```

Comment:
Yes, you can look a few line below ! (Just added an error case on DSC which cannot be managed with it)